google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -67,6 +68,12 @@
  *     {@code java.log}).
  * <li>{@code com.google.cloud.logging.LoggingHandler.level} specifies the default level for the
  *     handler (defaults to {@code Level.INFO}).
+ *     <p> This configuration also sets the "base severity level".
```

Comment:
I agree this might be a lot of details most users won't care about. On the other hand, I felt it might be useful to document, since it represents a sizable percentage change in RPC efficiency. Of course, most users will probably not log enough to see a big difference in absolute terms.

I'll be happy to delete this if you feel strongly about it.